README.md

```diff
@@ -1,2 +1,3 @@
 # Kunpeng
 Welcome to Kunpeng compute community.
+Join the [ARM Server Ecosystem slack channel](https://join.slack.com/t/armserverecosystem/shared_invite/enQtOTE0MDMxOTc0MTY0LTBiMTdkZWFhMjZmYzI2ZWVmYWUxMTU1YTcxY2NlZWViOGM5YTY4YzkwZDU3M2ZiZWUxMDQzMmU0NGY5YmFiYWY) to discuss all things related to ARM servers with experts all over the world.
```

Comment:
Look like we need a extra new line between l2 and l3, otherwise, the markdown will put these two lines into one line.